assets/airports/airportLoadList.js

```diff
@@ -375,12 +375,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             name: 'Bucharest Henri Coanda International Airport',
             premium: false
         },
-        // {
-        //     icao: 'loww',
-        //     level: 'medium',
-        //     name: 'Vienna International Airport',
-        //     premium: false
-        // },
+        {
+            icao: 'loww',
+            level: 'medium',
```

Comment:
If I counted correctly, there are almost 39acph at the moment, which would make this technically 'easy'. You can either change the level to easy or find somewhere to distribute the extra 2.0976acph needed to make this officially 'medium'